specificatie/gba-openapi.yaml

```diff
@@ -226,7 +226,7 @@ paths:
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/develop/specificatie/common.yaml#/components/responses/default"
       tags:
         - BRP historie bevragen
-# Daar waar mogelijk wordt nu verwezen naar componenten die in HC BRP Personen zijn gedefinieerd. Dat is echter alleen mogelijk als betreffende componenten zelf niet ook een $ref bevatten. 
+# Daar waar mogelijk wordt nu verwezen naar componenten die in HC BRP Personen zijn gedefinieerd. Dat is echter alleen mogelijk als betreffende componenten zelf niet ook een $ref bevatten.
 # Daar waar dat wel het geval is zijn de componenten in de openapi.yaml van HC BRP Historie gekopieerd.
```

Comment:
Kan  misschien korter: 
Waar mogelijk wordt verwezen naar componenten uit HC BRP Personen. Dat kan alleen als deze zelf niet ook een $ref bevatten. In dat geval zijn de componenten in de openapi.yaml van HC BRP Historie gekopieerd.

Reply:
Doen we dat nog, verwijzen naar componenten uit de BRP Personen repo?

Reply:
Op enkele plaatsen wordt er in de OAS nog verwezen naar de Haal-Centraal-BRP-bevragen repository. Dat gaat om de GBAdatum en enkele foutmeldingen. Die zitten echter ook in de lokale common van BRP-historie-bevragen.

Zal ik dat aanpassen zodat er geen enkele verwijzing naar een andere repository meer is ? Dat is volgens mij wat we willen. Ook het hier genoemde commentaar zal ik dan verwijderen.

Wat mij betreft kan dat in PR #87 mee